packages/microfrontend-sdk/src/types.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export enum EventsTypes {
+  SET_SELECTED_PROJECT = 'SET_SELECTED_PROJECT',
+  HIDE_ENVIRONMENT_SELECTION = 'HIDE_ENVIRONMENT_SELECTION',
+  SHOW_ENVIRONMENT_SELECTION = 'SHOW_ENVIRONMENT_SELECTION',
+  // Deprecated, probably used only from Visualize
+  GET_CONFIG = 'GET_CONFIG',
```

Comment:
let's not add deprecated things to the new sdk, Visualize will need to be improved to adopt the sdk (as for fast data)

Reply:
removed